x-pack/plugins/index_management/public/application/components/mappings_editor/lib/utils.ts

```diff
@@ -39,13 +39,23 @@ import { TreeItem } from '../components/tree';
 
 export const getUniqueId = () => uuidv4();
 
+const fieldsWithoutMultiFields: DataType[] = [
```

Comment:
From my testing, I _think_ `rank_features` and `percolator` should also be included in this list.

<img width="1336" alt="Screenshot 2024-03-07 at 12 43 53 PM" src="https://github.com/elastic/kibana/assets/5226211/f3438b09-b69e-4d42-a7ae-79b8d7c6dc40">